metricbeat/module/kubelet/container/container.go

```diff
@@ -41,7 +41,7 @@ type MetricSet struct {
 // Part of new is also setting up the configuration by processing additional
 // configuration entries if needed.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Warn("BETA: The kubelet container metricset is alpha")
+	logp.Warn("BETA: The kubelet container metricset is beta")
```

Comment:
We recently added `logp.Beta(...)` which makes it easier for us to find. Lets switch to this one.